ghcide/src/Development/IDE/Core/PluginUtils.hs

```diff
@@ -0,0 +1,132 @@
+{-# LANGUAGE GADTs #-}
+module Development.IDE.Core.PluginUtils where
```

Comment:
should also be in `hls-plugin-api`?

Reply:
mostly we are rewrapping around functions in gchide, and because hls-plugin-api can't import ghcide, it makes sense to put these stuff here (not sure if there's a better name for this module though)

Reply:
Oh right, because of the shake stuff, got it